pkgs/development/python-modules/django-extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv, buildPythonPackage, fetchFromGitHub,
+  vobject, mock, tox, pytestcov, pytest-django, pytest, shortuuid, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-extensions";
+  version = "1.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "08rd9zswvjb9dixzyd3p3l3hw3wwhqkgyjvid65niybzjl1xdb5h";
+  };
+
+  buildInputs = [ django vobject mock tox pytestcov pytest-django pytest shortuuid];
```

Comment:
For python they have the following semantics:
- `propagatedBuildInputs`: all python packages, which should be available for consumers of this package or in other words runtime dependencies
- `buildInputs`: dependencies only needed to build the package, for python this is most often test dependencies like `mock` and `pytest`.
- `nativeBuildInputs`: if this package would be cross-compiled, host-native packages like compilers would be included here, however it is not that common for python packages to require them.